### DISCUSSION TITLE
How to configure sass loader in nuxt config?

### DISCUSSION THREAD

**Discussion by chesterlaykin** [nuxt/nuxt#10655] [Questions (Nuxt 2)]
Hi!
I'm using nuxt 2 and I wanted to include the css through sass loader, to enable sass variables and mixins etc in vue components.

In a regular wepack config you could import scss in a property called data inside "options":
```
{
              loader: "sass-loader",
              options: {
                data: '@import "./assets/styles/variables/variables.scss";',
                outputStyle: "compressed"
          }
   }
```

In nuxt config there are supposed to be two ways to customize loader options ( [configuration build - loaders](https://nuxtjs.org/docs/configuration-glossary/configuration-build/#loaders) )

You're supposed to be able to add the loader like this:  `sass: {}`
But this didn't work:
```
  sass: {
    options: {
      data: '@import "@/assets/scss/start-files/index.scss";',
    }
  },
```
Nor this:
```
  sass: {
      data: '@import "@/assets/scss/start-files/index.scss";',
      hello: false
  },
```
(and it doesn't validate it, so I can add my own properties, nothing happens)

The other way is to use **build.extend.**
I found a [post](https://github.com/nuxt/nuxt.js/issues/772) regarding it, but that's probaby for webpack 3 or earlier.

It's complicated to loop through the loaders , but after adding the data property, it didn't work for me anyway:
```
 build: {
    extend(config, { isDev, isClient }) {
      const sassLoader = config.module.rules.find((rule) => {
        return rule.test.toString().includes("sass");
      });
      sassLoader.oneOf.forEach((item) => {
        const loader = item.use.find((obj) => {
          return obj.loader.toString().includes("sass-loader");
        });
        loader.options.data = '@import "@/assets/scss/start-files/index.scss";';
      });
    },
  }
```
So how will I be able to add this config option??

**Comment by beshur** [comment]
did you fix it?
